src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIncentiveReviewService.kt

```diff
@@ -176,26 +176,16 @@ class PrisonerIncentiveReviewService(
       val iepLevel = getIncentiveLevelForReviewType(prisonerInfo, reviewType)
       val comment = getReviewCommentForEvent(prisonOffenderEvent)
 
-      val createIncentiveReviewRequest = CreateIncentiveReviewRequest(
-        iepLevel = iepLevel,
-        comment = comment,
-        reviewType = reviewType,
-      )
-
-      val locationInfo = prisonApiService.getLocationById(prisonerInfo.assignedLivingUnitId, true)
```

Comment:
This is using the client credentials which may explain why in AppInsights some requests seems to come from the UI ([see tracking spreadsheet](https://docs.google.com/spreadsheets/d/1xa0MJNMreQcqONtS9alkF_5Hg3NvWSCckDIyr2KQOXA/edit?gid=0#gid=0)).